lib/index.js

```diff
@@ -141,7 +141,12 @@ export const middleware = store => next => async action => {
   let price;
 
   switch (action.type) {
+    case 'SET_LOADING':
     case 'STATE_REFRESHED': {
+      // can skip if node is currently loading
+      // if it's done loading then update feeds
+      if (action.type === 'SET_LOADING' && action.payload) break;
```

Comment:
LGTM but do you need to test for `action.payload`? why would that be false?

Reply:
Ha ok took me 5 days to figure this out too :-P
If `action.payload` is TRUE (app is still loading) then the `switch` statement will `break`, meaning the price will not update. When `SET_LOADING` is finally false (app done loading) execution continues, and price gets updated. So this is all right! Thank you